src/lib/routes/unleash-types.ts

```diff
@@ -3,4 +3,6 @@ import User from '../user';
 
 export interface IAuthRequest extends Request {
```

Comment:
Ah, clever idea, maybe we can put `AuthedRequest { user: User }` inside src/types/core.ts ?